packages/react/src/ReactLazy.js

```diff
@@ -134,53 +135,34 @@ export function lazy<T>(
     _init: lazyInitializer,
   };
 
-  if (__DEV__) {
-    // In production, this would just set it on the object.
-    let defaultProps;
-    let propTypes;
-    // $FlowFixMe[prop-missing]
-    Object.defineProperties(lazyType, {
-      defaultProps: {
-        configurable: true,
-        get() {
-          return defaultProps;
-        },
-        // $FlowFixMe[missing-local-annot]
-        set(newDefaultProps) {
-          console.error(
-            'It is not supported to assign `defaultProps` to ' +
-              'a lazy component import. Either specify them where the component ' +
-              'is defined, or create a wrapping component around it.',
-          );
-          defaultProps = newDefaultProps;
-          // Match production behavior more closely:
-          // $FlowFixMe[prop-missing]
-          Object.defineProperty(lazyType, 'defaultProps', {
-            enumerable: true,
-          });
-        },
-      },
-      propTypes: {
```

Comment:
Unrelated, but I removed this warning entirely since `propTypes` support has already been removed from the codebase.